src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Bit.BlazorUI.Playground;
+
+public static class StringExtensions
+{
+    public static bool HasValue(this string? value, bool ignoreWhiteSpace = true)
+    {
+        return ignoreWhiteSpace
+            ? string.IsNullOrWhiteSpace(value) is false
+            : string.IsNullOrEmpty(value) is false;
+    }
+
+    public static bool HasNoValue(this string? value, bool ignoreWhiteSpace = true)
+    {
+        return ignoreWhiteSpace
+            ? string.IsNullOrWhiteSpace(value)
+            : string.IsNullOrEmpty(value);
+    }
+}
```

Comment:
again, instead of redefining this class use `[assembly: InternalsVisibleTo("Bit.BlazorUI.Tests")]`